src/containers/Review/assignment/AssignAll.tsx

```diff
@@ -30,16 +33,25 @@ const AssignAll: React.FC<AssignAllProps> = ({ assignments, setReviewerForAll, c
       <Label className="uppercase-label" content={strings.ASSIGN_ALL_TO} />
       <Dropdown
         className="reviewer-dropdown"
-        options={options}
-        placeholder={strings.ASSIGN_ALL_PLACEHOLDER}
-        value={selected}
-        clearable
-        onChange={(_, { value }) => {
-          const reviewerId = value !== '' ? options[value as number].reviewerId : 0
-          setReviewerForAll(reviewerId)
-          setSelected(value as number | string)
-        }}
-      />
+        text={
+          options.find(({ value }) => value === selected)?.text || strings.ASSIGNMENT_NOT_ASSIGNED
+        }
```

Comment:
The previous `value={selected}` wasn't working after changing to use the header ASSIGNMENT_NOT_ASSIGNED